src/assets/stylesheets/palette/_scheme.scss

```diff
@@ -105,6 +105,16 @@
     img[src$="#only-dark"] {
       display: initial;
     }
+
+    // Hide images for github light mode
+    img[src$="#gh-light-mode-only"] {
```

Comment:
Please merge this with the rule in line 105 and above like this:

``` css
img[src$="#only-light"],
img[src$="#gh-light-mode-only"] {
  display: initial;
}

img[src$="#only-dark"],
img[src$="#gh-dark-mode-only"] {
  display: initial;
}
```

Reply:
done